content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: Maintenance Mode for Cluster Nodes
+description: Prepare node for maintenance, and restore it to previous state
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+From time to time, a node requires maintenance, for reasons relating to
+your organization, not to the Redis Enterprise cluster. The Maintenance
+Mode is a process to simplify the process of moving Redis shards away
+from node, and returning them after the maintenance is done.
+
+## Basic Usage
+
+Connect via SSH to the cluster, and use rladmin:
+
+```src
+$ rladmin node <id> maintenance_mode on
+```
+
+Wait for cluster to remove all Redis shards from node, and perform
+maintenance. Once maintenance is done, use the following to return the
+shards that were on the node back:
+
+```src
+$ rladmin node <id> maintenance_mode off
+```
+
+## Maintenance On Steps
+
+1. Checks that node with that id exists in the cluster.
+1. Checks that cluster will not lose quorum if node is shut off.
```

Comment:
@RoeyPrat What is the impact? Will this prevent maintenance?

Reply:
yes. maintenance will stop and print an error.